pkg/storage/azure/azure.go

```diff
@@ -292,7 +301,7 @@ func (d *driver) storageAccountsClient(cfg *Azure, environment autorestazure.Env
 		storageAccountsClient.Authorizer = d.authorizer
 	} else {
 		clientCredentialsConfig := auth.NewClientCredentialsConfig(cfg.ClientID, cfg.ClientSecret, cfg.TenantID)
```

Comment:
@jsafrane to my untrained eye it looks like this is reading the creds from the config which would be a problem with openshift/installer#5138

Reply:
these creds are from cloud-credential-operator, the operator requests them via https://github.com/openshift/cluster-image-registry-operator/blob/master/manifests/01-registry-credentials-request-azure.yaml

Reply:
Perfect. Thank you.